.github/workflows/stale_prs.yml

```
@@ -11,8 +11,10 @@ jobs:
     - uses: actions/stale@v1
       with:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
-        stale-pr-message: "This PR has been inactive for long enough to be automatically marked as stale. This means it is at risk of being closed by a maintainer if it is not updated or reviews are not addressed. If your PR is closed as stale, feel free to open a new one after dealing with the issues. This may also be an indication that the maintainers do not have interest in this change, you can try to convince them otherwise, or persist in the doomed world you have created."
+        stale-pr-message: "This PR has been marked as stale due to being in an unmergable state for 7 days. Please resolve any conflicts and add testing evidence, then contact a project maintainer to have the stale label removed."
         stale-pr-label: 'Stale'
-        exempt-pr-label: 'Needs Review'
-        days-before-stale: 100000
+        only-pr-labels: 'Stale,Needs Testing Evidence,Merge Conflict'
```

`only` means it requires **ALL** of the tags to be present. Any means any of the listed tags. Stale should not be included here, since it gets added when this condition is met.

```suggestion
        any-of-pr-labels: 'Needs Testing Evidence,Merge Conflict'
```

---

Oh wow thanks, that was a silly mistame